npm/pkg/transport/events_client.go

```diff
@@ -81,11 +90,12 @@ func (c *EventsClient) run(ctx context.Context, stopCh <-chan struct{}) error {
 		default:
 			if connectClient == nil {
 				klog.Info("Reconnecting to gRPC server controller")
-				opts := []grpc.CallOption{grpc.WaitForReady(true)}
+				opts := []grpc.CallOption{grpc.WaitForReady(false)}
```

Comment:
curious what this represents and why we had it true before, false now

Reply:
True blocks the connection until the server is ready to accept the connection. We don't want to block our connection since the server pod could crash which we are connecting or in the load-balancing case reject the connection in which case the agent would need to block until timeout. Instead we try sending, error out and try reconnecting.